platform/broadcom/rules.dep

```diff
@@ -30,3 +30,4 @@ include $(PLATFORM_PATH)/libsaithrift-dev.dep
 include $(PLATFORM_PATH)/docker-syncd-brcm-dnx.dep
 include $(PLATFORM_PATH)/docker-syncd-brcm-dnx-rpc.dep
 include $(PLATFORM_PATH)/../components/docker-gbsyncd-credo.dep
+include $(PLATFORM_PATH)/../components/docker-gbsyncd-broncos.dep
```

Comment:
It is not desirable to include and build all gbsyncd dockers by default, with more phy chips support. Could we optionally include it with a config variable like `INCLUDE_GBSYNCD_BRONCOS`? For example:

```
ifeq ($(INCLUDE_GBSYNCD_BRONCOS), y)
include $(PLATFORM_PATH)/../components/docker-gbsyncd-broncos.dep
endif
```